### PR TITLE
chore(docs): added CDN usage instruction

### PR DESCRIPTION
**Pull request by g-cheishvili**
## Related Issue
Part of SAP/fundamental-styles#3095

## Description
Added instructions of how to use [unpkg](https://unpkg.com/) for including fundamental-styles via CDN

### CODE REVIEW

**Review by Betrozov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![Slack](https://img.shields.io/badge/slack-ui--fundamentals-blue.svg?logo=slack)](https://join.slack.com/t/ui-fundamentals/shared_invite/enQtNTIzOTU0Mzc2NTc5LWQzZWI5MWFhYjE5OTc4YzliN2JhOTc1ZjQxZTg1YjZiMWZiYzRkNjMwYzgyMmFkYmNhZDVjMWE5MDIzOWEzMmM)
 [![REUSE status](https://api.reuse.software/badge/github.com/SAP/fundamental-styles)](https://api.reuse.software/info/github.com/SAP/fundamental-styles)
 <a href="https://storybook.js.org/">
-  <img src="https://raw.githubusercontent.com/storybookjs/brand/master/badge/badge-storybook.svg" />
+<img src="https://raw.githubusercontent.com/storybookjs/brand/master/badge/badge-storybook.svg" />
```
Comment (InnaAtanasova) [inline README.md:8]: Is it possible to get rid of the word `master`?

**Review by InnaAtanasova — COMMENTED** [review]
@ynnenu could you take a look at this PR?

File: README.md (hunk 1 above)
Comment (g-cheishvili) [reply README.md]: Changed it with `main`

File: README.md [hunk 2]
```diff
@@ -14,157 +14,210 @@
 
 ## What is Fundamental Library Styles?
 
-Fundamental Library Styles is a light-weight presentation layer that can be used with your UI framework of choice (e.g. Angular, React, Vue, etc.). With Fundamental Library Styles, library of stylesheets and HTML tags, developers can build consistent Fiori apps in any web-based technology.
+Fundamental Library Styles is a light-weight presentation layer that can be used with your UI framework of choice (e.g.
+Angular, React, Vue, etc.). With Fundamental Library Styles, library of stylesheets and HTML tags, developers can build
+consistent Fiori apps in any web-based technology.
```
Comment (ynnenu) [inline README.md:19]: Fundamental Library Styles is a light-weight presentation layer that can be used with **developers'** UI framework of choice (e.g.Angular, React, Vue, etc.) 

I suggest developers' in order to stay consistent with the next sentence. 

As per the second sentence, I would suggest avoiding starting the sentence with "With".  Maybe the sentence can be rewritten:
Developers can build consistent Fiori apps in any web-based technology, by combining Fundamental Styles, library of stylesheets and HTML tags.

File: README.md [hunk 3]
```diff
@@ -14,157 +14,210 @@
...
 ## Getting Started
+
 The library is modular so you can use as little or as much as you need.
 
 ### CDN
-The fully compiled, minified library is available via CDN for inclusion in your application.
 
+The fully compiled, minified library is available via [unpkg CDN](https://unpkg.com/) for inclusion in your application.
+
+For `prerelease` version use
```
Comment (ynnenu) [inline README.md:34]: For `prerelease` version use:

File: README.md [hunk 4]
```diff
@@ -14,157 +14,210 @@
...
 npm install fundamental-styles --save
 ````
 
-> NOTE: We only distribute compiled CSS for each component, not the full project or HTML for specific components. 
+> NOTE: We only distribute compiled CSS for each component, not the full project or HTML for specific components.
 
 ### Distributed CSS Files
 
-The distrubtion folder contains several css files per component: a default file and one file per supported theme (`sap_fiori_3`, `sap_fiori_3_dark`, `sap_fori_3_light_dark`, `sap_fiori_3_hcb`, `sap_fori_3_hcw`, `sap_horizon`). Each themed file includes fallback support for css variables in IE11 for the corresponding theme. The default file includes fallbacks for `sap_fiori_3`. 
+The distrubtion folder contains several css files per component: a default file and one file per supported
```
Comment (ynnenu) [inline README.md:73]: The distrubtion folder contains several CSS files per component:

File: README.md [hunk 5]
```diff
@@ -14,157 +14,210 @@
...
+theme (`sap_fiori_3`, `sap_fiori_3_dark`, `sap_fori_3_light_dark`, `sap_fiori_3_hcb`, `sap_fori_3_hcw`, `sap_horizon`).
+Each themed file includes fallback support for css variables in IE11 for the corresponding theme. The default file
```
Comment (ynnenu) [inline README.md:75]: Each themed file includes fallback support for **CSS** variables in IE11..

File: README.md [hunk 6]
```diff
@@ -14,157 +14,210 @@
...
-        font-weight: normal;
-        font-style: normal;
-    }
-
-    html {
-      font-size: 16px;
-    }
+
+This project does not contain fonts and icons - they must be added to your project separately. Download the @sap-theming
+library. After adding fonts and icons to your project, include the following in your css:
```
[the remark this answers is not in the document]
Comment (g-cheishvili) [reply README.md:87]: Uppercase "css"?

File: README.md [hunk 7]
```diff
@@ -14,157 +14,210 @@
...
+The `fundamental-styles` library follows [Semantic Versioning](https://semver.org/). These components strictly adhere to
+the `[MAJOR].[MINOR].[PATCH]` numbering system (also known as `[BREAKING].[FEATURE].[FIX]`).
 
-Merges to the `main` branch will be published as a prerelease. Pre-releases will include an **rc** version (_e.g._ `[MAJOR].[MINOR].[PATCH]-rc.[RC]`).
+Merges to the `main` branch will be published as a prerelease. Pre-releases will include an **rc** version (_
+e.g._ `[MAJOR].[MINOR].[PATCH]-rc.[RC]`).
 
 The following circumstances will be considered a MAJOR or BREAKING change:
+
 * Dropping existing classnames, css variables, color names, color groups, spacing parameters
```
Comment (ynnenu) [inline README.md:209]: CSS variables

File: README.md [hunk 8]
```diff
@@ -14,157 +14,210 @@
...
 * The existing underlying HTML markup of a component is altered
 * Non-visual HTML attribute changes/additions (such as `role`, `aria-*`, `data-*`)
   > Note: Fundamental Styles provides CSS directly, and HTML as reference to consumers. Because of the reference relationship of the HTML seen in Fundamental Styles, we want to be very clear when we alter that reference so that it is properly reflected in JS implementation libraries. Because of this, even non-visual changes will be treated as breaking.
 
 The following circumstances will NOT be considered a MAJOR or BREAKING change:
-* Introducing new  classnames, css variables, color names, color groups, spacing parameters
+
+* Introducing new classnames, css variables, color names, color groups, spacing parameters
```
Comment (ynnenu) [inline README.md:216]: CSS

**Review by droshev — APPROVED** [review]

**Pull request merged**